### PR TITLE
Add Chinese (Traditional) language, fixed Chinese (Simplified) language

### PR DESCRIPTION
**Pull request by NightFurySL2001**
Updates for both Chinese translation.

P/S: is it possible to allow more than one locale tag for one language? E.g. allow `cn`, `zh-cn`, `zh-hans`, `zhs` to point to the same translation string instead of using more than one line to store it in file. (Currently I added `zhs` to `cn`, also added `tw` and `zht` for Traditional Chinese)

### CODE REVIEW

File: docs/readme_cn.md [hunk 1]
```diff
@@ -55,44 +57,46 @@
...
+> 参数：`&hide=stars,commits,prs,issues,contribs`
 
 ```md
 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=contribs,prs)
 ```
 
 ### 将私人项目贡献添加到总提交计数中
 
-你可以使用参数 `?count_private=true` 把私人贡献计数添加到总提交计数中。
+你可以使用参数 `?count_private=true` 参数把私人贡献计数添加到总提交计数中。
```
Comment (Kotoki1337) [inline docs/readme_cn.md:97]:
```suggestion
你可以使用 `?count_private=true` 参数把私人贡献计数添加到总提交计数中。
```
typo?

File: docs/readme_cn.md [hunk 2]
```diff
@@ -55,44 +57,46 @@
...
 
-_注：如果你是自己部署本项目，私人贡献将会默认被计数，如果不是自己部署，你需要分享你的私人贡献计数。_
+_注：如果你是自行部署本项目，私人贡献将会默认被计数，如果不是自己部署，你需要分享你的私人贡献计数。_
```
Comment (Kotoki1337) [inline docs/readme_cn.md:99]:
```suggestion
_注：如果你是自行部署本项目，私人贡献将会默认被计数，如果不是自行部署，你需要分享你的私人贡献计数。_
```

File: docs/readme_cn.md [hunk 3]
```diff
@@ -299,18 +362,18 @@ _注意：热门语言并不表示我的技能水平或类似的水平，它是
...
 因为 GitHub 的 API 每个小时只允许 5 千次请求，我的 `https://github-readme-stats.vercel.app/api` 很有可能会触发限制。如果你将其托管在自己的 Vercel 服务器上，那么你就不必为此担心。点击 deploy 按钮来开始你的部署！
 
 注意: 从 [#58](https://github.com/anuraghazra/github-readme-stats/pull/58) 开始，我们应该能够处理超过 5 千次的请求，并且不会出现宕机问题 :D
 
-[![Deploy to Vercel](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/anuraghazra/github-readme-stats)
+[![在 Vercel 上部署](https://vercel.com/button)](https://vercel.com/import/project?template=https://github.com/anuraghazra/github-readme-stats)
 
 <details>
- <summary>设置 Vercel 的指导</summary>
+ <summary>部署 Vercel 指导</summary>
```
Comment (Kotoki1337) [inline docs/readme_cn.md:376]:
```suggestion
 <summary>Vercel 部署指南</summary>
```
原文为 `guide`

File: docs/readme_cn.md [hunk 4]
```diff
@@ -258,37 +313,45 @@ _注意：热门语言并不表示我的技能水平或类似的水平，它是
...
-- 渐变
+- 渐变背景
 
 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&bg_color=30,e96443,904e95&title_color=fff&text_color=fff)
 
 - 自定义统计卡片
 
 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api/?username=anuraghazra&show_icons=true&title_color=fff&icon_color=79ff97&text_color=9f9f9f&bg_color=151515)
 
+- 设置卡片在地化语言（简中请用 `zhs` 或 `cn`，繁中请用 `zht` 或 `tw`）
```
Comment (Kotoki1337) [inline docs/readme_cn.md:334]:
```suggestion
- 设置卡片本地化语言（简中请用 `zhs` 或 `cn`，繁中请用 `zht` 或 `tw`）
```

File: docs/readme_cn.md [hunk 5]
```diff
@@ -198,35 +219,51 @@ GitHub 更多置顶 允许你在使用 GitHub readme profile 时，在个人资
...
 [![Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats&show_owner=true)](https://github.com/anuraghazra/github-readme-stats)
 
-# 热门语言卡片
+# 常用语言卡片
 
-热门语言卡片显示了 GitHub 用户常用的编程语言。
+常用语言卡片显示了 GitHub 用户常用的编程语言。
 
-_注意：热门语言并不表示我的技能水平或类似的水平，它是用来衡量用户在 github 上拥有最多代码的语言的一项指标，它是 github-readme-stats 的新特性_
+_注意：常用语言并不代表我的技能水平或类似的资讯，它是用来衡量用户在 github 上拥有最多代码的语言的一项指标，它是 github-readme-stats 的新特性。_
```
Comment (Kotoki1337) [inline docs/readme_cn.md:230]:
```suggestion
_注意：常用语言并不代表我的技能水平或类似的信息，它是用来衡量用户在 github 上拥有最多代码的语言的一项指标，它是 github-readme-stats 的新特性。_
```

File: docs/readme_cn.md (hunk 1 above)
Comment (NightFurySL2001) [reply docs/readme_cn.md:97]: Yes, thanks for catching

**Pull request closed without merging**